### ISSUE TITLE
Build failed

### ISSUE THREAD

**Issue by arnwas** [wbsoft/frescobaldi#1104]
Build from current git failed, hinting to this highly complex patch proposal.

Cheers!

```
diff --git a/frescobaldi_app/file_import/abc.py b/frescobaldi_app/file_import/abc.py
index a090142e..8a6d659e 100644
--- a/frescobaldi_app/file_import/abc.py
+++ b/frescobaldi_app/file_import/abc.py
@@ -88,7 +88,7 @@ class Dialog(toly_dialog.ToLyDialog):
             j.stdout = lambda: content
             os.remove(temp_file)
         else:
-            j.stdout = lambda: (_('Failed to import ABC')
+            j.stdout = lambda: (_('Failed to import ABC'))
         return j
 
     def loadSettings(self):
```

**Comment by uliska** [comment]
Thanks for the report and suggestion.
Do you want permanent credit for it? Then include the edit directly in the file, letting Github create a Pull Request for you.
Otherwise I'll simply apply the fix.

**Comment by arnwas** [comment]
Done. Thanks!

**Comment by uliska** [comment]
Fixed in #1107
